document/CoreFormat.mdk

```diff
@@ -2439,9 +2440,35 @@ C++: `"std::runtime_error"`
 C#: `"System.ArgumentNullException"`
 ~
 
+If the tool does not have access to information about the object that was thrown, the `type` property shall be absent.
+
+~ IsoNote
+This will be the case in C++ programs that throw objects that do not derive from `std::exception`:
```

Comment:
If I'm in a catch, could I do this? Since my C++ fu is weak I'll write in a nasty mixture of C#/C++:

```
catch (std::runtime_error ex)
{
    logNotification("std::runtime_error", ex.what());
}
```